.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,4 @@
+contact_links:
+  - name: Usage question
+    url: https://neurostars.org/tags/c/software-support/234/babs
```

Comment:
Hmmm that page does not exist... Maybe `234` is for xcp-d? What I should do to make a page for BABS - create a new question on neurostars?

Reply:
234 is for all software-support questions. The problem is that there are no babs-tagged questions yet. I've got some permissions on NeuroStars, so I just created one. The link should work now.

Reply:
Perfect! Thanks a lot! I'll merge your changes to main after I add to docs that users can ask questions on neurostars!

Reply:
Okay based on Matt's thoughts on 9/15/2023 ("it's a bit early for neurostars") in the BABS slack channel, I think I'll remove the option of neurostars and this `config.yml` file for now.